apps/backend/src/config/config.service.ts

```diff
@@ -51,7 +55,8 @@ export class ConfigService {
       enabledOAuth: this.enabledOauthStrategies(),
       oidcName: this.get('OIDC_NAME') || '',
       ldap: this.get('LDAP_ENABLED')?.toLocaleLowerCase() === 'true' || false,
-      registrationEnabled: this.isRegistrationAllowed()
+      registrationEnabled: this.isRegistrationAllowed(),
+      localLoginEnabled: this.isLocalLoginAllowed()
```

Comment:
Since `REGISTRATION_DISABLED` already follows this format, going against this would break existing deployments and it would be confusing to have both here.